.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        pg_major: ["9.6", "10.9", "11.4", "12.2"]
```

Comment:
It seems a little counter-intuitive to make the PostgreSQL version the primary matrix variable, as opposed to the PostGIS version, but that's the only thing that's actually unique per run. Turns out GitHub Actions won't add multiple includes to the same matrix match, i.e. trying to apply two `pg_major` variables to the same `postgis_major` variable in the matrix; it'll just use the first include.

I was able to test this quickly using [`act`](https://github.com/nektos/act), a CLI tool for testing workflows locally. Dry run mode (`act -n`) is handy for making sure your matrix is correct.

Reply:
> I was able to test this quickly using act, a CLI tool for testing workflows locally. Dry run mode (act -n) is handy for making sure your matrix is correct.

Nice. I found this tool useful as well when testing non-PR related events (e.g., issue creation).

Reply:
> I was able to test this quickly using act, a CLI tool for testing workflows locally. Dry run mode (act -n) is handy for making sure your matrix is correct.

Nice. I found this tool useful as well when testing non-PR related events (e.g., issue creation).